sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryCompilationErrors.scala

```diff
@@ -1269,7 +1278,7 @@ private[sql] object QueryCompilationErrors extends QueryErrorsBase {
 
   private def notSupportedInJDBCCatalog(cmd: String): Throwable = {
     new AnalysisException(
-      errorClass = "_LEGACY_ERROR_TEMP_1119",
+      errorClass = "NOT_SUPPORTED_OPERATION_IN_JDBC_CATALOG",
       messageParameters = Map("cmd" -> cmd))
```

Comment:
because we have some arbitrary text after SQL statements like:
```sql
CREATE NAMESPACE with property $k
```
If so, how about to add sub-classes the commands related to properties?

Reply:
How about such an approach?
```json
"NOT_SUPPORTED_IN_JDBC_CATALOG" : {
  "message" : [
    "Not supported operation in JDBC catalog:"
  ],
  "subClass" : {
    "OPERATION" : {
      "message" : [
        "<cmd>"
      ]
    },
    "OPERATION_WITH_PROPERTY" : {
      "message" : [
        "<cmd> with property <property>."
      ]
    }
  },
  "sqlState" : "46110"
}
```
(Also I'm not sure if the sql state is the correct one)

And corresponding error messages examples: 

- `[NOT_SUPPORTED_IN_JDBC_CATALOG.OPERATION_WITH_PROPERTY] Not supported operation in JDBC catalog: SET NAMESPACE with property "location".`
- `[NOT_SUPPORTED_IN_JDBC_CATALOG.OPERATION] Not supported operation in JDBC catalog: CREATE NAMESPACE ... LOCATION ...`

Reply:
Could you replace `OPERATION` by `COMMAND` since we use the last word mostly.

> "sqlState" : "46110"
> (Also I'm not sure if the sql state is the correct one)

SGTM

Reply:
Done. Also added `"sqlState" : "46110"` for `NOT_SUPPORTED_COMMAND_FOR_V2_TABLE`